### ISSUE TITLE
Acid Rain will not disappear 

### ISSUE THREAD

**Issue by ladyshea** [TeamTwilight/twilightforest#644]
Forge version: 1.12.2
Twilight Forest version:  Twilight Forest - MC 1.12.2 - 3.8.654

What I expected to happen: 
Acid rain to disappear after I killed the Ur-Ghast, Hydra, and Snow Queen. I even have the achievement telling me I can enter the highlands now, but still nothing has changed. 

![image](https://user-images.githubusercontent.com/44149868/46926519-f9310580-cfe5-11e8-9822-2afe3b7790d3.png)

What happened instead: 
Acid rain did not go away. and I cannot enter the highlands to complete the progression system

**Comment by ladyshea** [comment]
So I figured out that it was a error with the thorns. Every time I entered the thorn protected area it was telling me that it was acid rain (complete with rain particles), and the little blue goblin came running out with the book telling me to go defeat the Hydra, Snow Queen, and Ur-Ghast. However the Troll caves were available to me and once I picked up the lamp the acid rain over the thorns area disappeared. 

Now I am able to complete the progression system
